components/button/button.css

```diff
@@ -114,13 +99,18 @@
   background-color: var(--primary-color);
 }
 
-.btn-secondary {
-  margin: 2rem;
+.btn {
   display: inline-flex;
-  align-items: center;
   justify-content: center;
-  box-sizing: border-box;
   position: relative;
+  box-sizing: border-box;
+  border: 0px;
+  margin: 0px;
```

Comment:
use just 0. px can be removed.

Reply:
any reason for this btw fixed it.

Reply:
For 0 any units won't make sense let it be anything unit. So it is recommended that we don't use any units with 0.